docs/ug/cli.md

```diff
@@ -261,3 +261,40 @@ Cannot be used with any other flags. This flag takes precedence over all other f
   Default: `./reposense-report`
 * Alias: `-v`
 * Example:`--view` or `-v`
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
```

Comment:
Seems like all CLI arguments are arranged in alphabetical order. Can follow the same for these 2 as well.